### PR TITLE
Add Services for Static Websites

### PR DESCRIPTION
**Pull request by aharris88**
I'm not sure if this is the best spot on the list, but I think it fits under front end development pretty well. Also #291 is a request for a list like this.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -136,6 +136,7 @@ Please read the [contribution guidelines](contributing.md) or the [creating a li
 - [jQuery](https://github.com/peterkokot/awesome-jquery)
 - [Web Audio](https://github.com/notthetup/awesome-webaudio)
 - [Offline-First](https://github.com/pazguille/offline-first)
+- [Services For Static Websites](https://github.com/aharris88/awesome-services-for-static-websites)
```
Comment (sindresorhus) [inline readme.md]: > Static Website Services

?

**Comment by sindresorhus** [comment]
Good start, but I would have liked to see more descriptions for the items.

**Comment by aharris88** [comment]
Ok, I'll work on that. Thanks.

File: readme.md (hunk 1 above)
Comment (aharris88) [inline readme.md]: Static Website Services would probably be better. I'll rename my repo as well.

**Comment by sindresorhus** [comment]
Closing this for now. Ping me when it's ready for reconsideration and I'll reopen.

**Comment by aharris88** [comment]
I added descriptions and made sure that everything on the list is awesome.

**Comment by aharris88** [comment]
I'm having trouble updating this pull request. I amended my commit and pushed to the branch [`patch2`](https://github.com/aharris88/awesome/commits/patch-2) and force pushed `git push -f`, but this pull request isn't updated...

**Comment by sindresorhus** [comment]
Yeah, seems like the PR gets borked if you do that when it's closed. Just submit a new one ;)

**Pull request closed without merging**